src/commands/ipfs.js

```diff
@@ -22,25 +21,17 @@ exports.task = ({ apmOptions }) => {
 	  	task: async (ctx, task) => {
 		    // If the dev manually set their IPFS node, skip install and running check
 		    if (apmOptions.ipfs.rpc.default) {
-		      const installed = await isIPFSInstalled()
-		      if (!installed) {
-		        setTimeout(() => opn('https://ipfs.io/docs/install'), 2500)
-		        throw new Error(`
-		          Running your app requires IPFS. Opening install instructions in your browser`
-		        )
-		      } else {
-		        const running = await isPortTaken(apmOptions.ipfs.rpc.port)
-		        if (!running) {
-		          task.output = 'Starting IPFS at port: ' + apmOptions.ipfs.rpc.port
-		          await startIPFSDaemon()
-		          ctx.started = true
-		          await setIPFSCORS(apmOptions.ipfs.rpc)
-		        } else {
-		          task.output = 'IPFS is started, checking CORS config'
-		          await setIPFSCORS(apmOptions.ipfs.rpc)
-		          return 'Connected to IPFS daemon ar port: '+ apmOptions.ipfs.rpc.port 
-		        }
-		      }
+					const running = await isPortTaken(apmOptions.ipfs.rpc.port)
+					if (!running) {
+						task.output = 'Starting IPFS at port: ' + apmOptions.ipfs.rpc.port
+						await startIPFSDaemon()
+						ctx.started = true
+						await setIPFSCORS(apmOptions.ipfs.rpc)
+					} else {
+						task.output = 'IPFS is started, checking CORS config'
+						await setIPFSCORS(apmOptions.ipfs.rpc)
+						return 'Connected to IPFS daemon ar port: '+ apmOptions.ipfs.rpc.port 
+					}
```

Comment:
I think this is little too far indented no?

Reply:
Indeed, my code editor replaced spaces with tabs. Fixed it now